sqlglot/dialects/bigquery.py

```diff
@@ -829,6 +836,50 @@ def _parse_features_at_time(self) -> exp.FeaturesAtTime:
 
             return expr
 
+        def _parse_export_data(self) -> exp.Export:
+            # https://cloud.google.com/bigquery/docs/reference/standard-sql/export-statements
+            if not self._match_text_seq("DATA"):
+                self.raise_error("Expected 'DATA' after 'EXPORT'")
+
+            with_connection = None
+            options = None
+
+            if self._match_text_seq("WITH", "CONNECTION"):
+                parts = []
+                while True:
+                    part = self._parse_var()
+                    if not part:
+                        break
+                    parts.append(part.name)
+                    if not self._match(TokenType.DOT):
+                        break
+
+                if not parts:
+                    self.raise_error("Expected connection name after WITH CONNECTION")
+
+                with_connection = exp.Identifier(this=".".join(parts))
+
+            if self._match_text_seq("OPTIONS"):
+                self._match(TokenType.L_PAREN)
+                options = self._parse_properties()
+                self._match(TokenType.R_PAREN)
+            else:
+                self.raise_error("Expected 'OPTIONS' after 'EXPORT DATA'")
+
+            self._match_text_seq("AS")
+
+            # Parse the full SELECT statement
+            query = self._parse_statement()
+            if not isinstance(query, exp.Select):
+                self.raise_error("Expected SELECT statement in EXPORT DATA")
+
```

Comment:
You don't have to do error handling like this, SQLGlot is intentionally lenient in these cases.